bundle/deploy/terraform/destroy.go

```diff
@@ -66,7 +19,7 @@ func (w *destroy) Name() string {
 func (w *destroy) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
 	// return early if plan is empty
 	if b.Plan.IsEmpty {
-		cmdio.LogString(ctx, "No resources to destroy in plan. Skipping destroy!")
+		cmdio.LogString(ctx, "No resources to destroy")
```

Comment:
If we're not logging when we are actually destroying resources, do we need to display this message if there aren't any?